docs/logging-usage.rst

```diff
@@ -0,0 +1,295 @@
+Using the API
+=============
+
+Authentication / Configuration
+------------------------------
+
+- Use :class:`Client <gcloud.logging.client.Client>` objects to configure
+  your applications.
+
+- :class:`Client <gcloud.logging.client.Client>` objects hold both a ``project``
+  and an authenticated connection to the Logging service.
+
+- The authentication credentials can be implicitly determined from the
+  environment or directly via
+  :meth:`from_service_account_json <gcloud.logging.client.Client.from_service_account_json>`
+  and
+  :meth:`from_service_account_p12 <gcloud.logging.client.Client.from_service_account_p12>`.
+
+- After setting ``GOOGLE_APPLICATION_CREDENTIALS`` and ``GCLOUD_PROJECT``
+  environment variables, create a :class:`Client <gcloud.logging.client.Client>`
+
+  .. doctest::
+
+     >>> from gcloud import logging
+     >>> client = logging.Client()
+
+Writing log entries
+-------------------
+
+Write a simple text entry to a log.
+
+.. doctest::
+
+   >>> from gcloud import logging
+   >>> client = logging.Client()
+   >>> log = client.log('log_name')
```

Comment:
I'm conflicted:  the API [refers to the resource type as `logs`](https://cloud.google.com/logging/docs/api/ref_v2beta1/rest/v2beta1/projects.logs), but the more natural name in Python land might be "logger" (except, of course, that it is not related remotely to the stdlib's `logger`).